docs/development/developer_guide.md

```diff
@@ -23,7 +23,7 @@ Install dependencies
 go mod tidy
```

Comment:
Do you think that it is a good idea to make it clear that the `GOPATH` variable needs to be set to the `go` workspace directory and that the `GOPATH` variable needs to be added to the beginning of the `PATH` variable (in case these are not already done)?

```
export GOPATH=/set/to/local/go/workspace/directory # e.g. /Users/tariqhasan/go
export PATH="$GOPATH/bin:$PATH"
```

Reply:
I don't think so. That is a part of the generic Go setup. So, the developer should follow the Go documentation.

Reply:
+1 to what @tenzen-y said